pkg/spec/types.go

```diff
@@ -189,6 +199,11 @@ type JobSpecMod struct {
 	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,conflicting,omitempty"`
 }
 
-type Controller struct {
-	Controller string `json:"controller,omitempty"`
+// Ochestrator: OpenShift
+// DeploymentConfigSpecMod is Kedge's extension of OpenShift DeploymentConfig in order to define and allow
+// a complete kedge app based on OpenShift
+// kedgeSpec: io.kedge.DeploymentConfigSpecMod
+type DeploymentConfigSpecMod struct {
+	ControllerFields                  `json:",inline"`
+	os_deploy_v1.DeploymentConfigSpec `json:",inline"`
```

Comment:
ping @surajssd, can you help with the comment for this field, or create an issue for it?